README.md

```diff
@@ -122,6 +122,22 @@ The following steps are for Linux and OSX.
     colcon build
     ```
 
+    If `colcon build` fails with [this issue](https://github.com/gazebosim/ros_gz/issues/401)
```

Comment:
if you run the `rosdep` command this should not happen, maybe a better option is to install this dependency via deps:

```bash
sudo apt-get install ros-<ROS_DISTRO>-actuators-msgs
```

Reply:
This is for a Tier 3 install of ROS Humble on Ubtuntu 20.04 so any of ros-humble-* won’t exist. Thus why the issue occurs even if you rosdep install